sdk/src/main/java/com/uid2/sdk/storage/SharedPreferencesStorageManager.kt

```diff
@@ -0,0 +1,35 @@
+package com.uid2.sdk.storage
+
+import android.content.Context
+import android.content.SharedPreferences
+import com.uid2.sdk.data.UID2Identity
+import org.json.JSONObject
+
+/**
+ * An implementation of the StorageManager that persists UID2Identity instances in clear-text via Shared Preferences.
+ */
+class SharedPreferencesStorageManager(private val context: Context) : StorageManager {
+
+    private val preferences: SharedPreferences by lazy {
+        context.getSharedPreferences(IDENTITY_KEY_FILE, Context.MODE_PRIVATE)
+    }
+
+    override fun saveIdentity(identity: UID2Identity) =
+        preferences.edit()
+            .putString(IDENTITY_PREF_KEY, identity.toJson().toString(0))
+            .commit()
+
+    override fun loadIdentity() = runCatching {
+        UID2Identity.fromJson(JSONObject(preferences.getString(IDENTITY_PREF_KEY, "")!!))
```

Comment:
Figure out a way NOT to use `!!` ..... ever! :)

Something like:
```
    overload fun loadIdentity() = preferences.getString("", "")?.let {
        runCatching { UID2Identity.fromJson(JSONObject(it)) }.getOrNull()
    }
```

Reply:
Makes sense, thanks